package.json

```diff
@@ -75,7 +75,8 @@
     "build-storybook": "build-storybook",
     "nx": "nx",
     "postinstall": "ngcc --properties es2015 browser module main --first-only --create-ivy-entry-points",
-    "docker-test": "docker run -u $(id -u) --rm -v \"$PWD\":/app trion/ng-cli-karma npm run test"
+    "docker-test": "docker run -u $(id -u) --rm -v \"$PWD\":/app trion/ng-cli-karma npm run test",
+    "build-local-app-shell": "npm run build core && npm run build app-shell && npm run build:schematics && npm run sync-versions"
```

Comment:
@valorkin please rebase, and there are already commands to build app shell. I sent it the other day to slack

Reply:
please see the build-deploy-library

Reply:
this one is faster and simpler

Reply:
Do you really need it? The difference between standard command and this extra one is like 15 sec.